src/stravalib/client.py

```diff
@@ -1429,10 +1445,13 @@ def get_segment_efforts(
             )
 
         if limit is not None:
+            warn_param_deprecation(limit, "date ranges")
```

Comment:
They have a per-page parameter but no limit. I noticed in our API that we use "limit" as a per-page parameter in a few places. But the way I read limit in the docs is it's a limit on how many items are returned. That feature doesn't work. Per page is still documented as a valid param.